core/templates/services/html-length.service.ts

```diff
@@ -108,15 +124,20 @@ export class HtmlLengthService {
 
   private getWeightForTextNodes(
       textNode: HTMLElement, calculationType: CalculationType): number {
-    const textContent = textNode.textContent || '';
+    const textContent = (textNode.textContent || '').trim();
+    return (textContent === '') ? 0 :
+        this.textWeightCalculation(textContent, calculationType);
```

Comment:
All method/function names should start with simple present-tense verbs.

Reply:
Done! Chnaged to calculateTextWeight()